pkg/controller/util.go

```diff
@@ -430,6 +427,12 @@ func MakeImporterPodSpec(image, verbose, pullPolicy string, podEnvVar *importPod
 	if len(pvc.OwnerReferences) == 1 {
 		ownerUID = pvc.OwnerReferences[0].UID
 	}
+
+	if podEnvVar.volumeMode == string(v1.PersistentVolumeBlock) {
+		pod.Spec.Containers[0].VolumeDevices = addVolumeDevices()
+	} else {
```

Comment:
Eventually block will need the scratch mount.
I guess in next PR when we will support non direct qcow, archive and registry

Reply:
correct